crafting_templates.xml

```diff
@@ -442,6 +442,7 @@
       <WeaponDescription id="crpg_OneHandedBastardSword" />
       <WeaponDescription id="crpg_TwoHandedSword" />
       <WeaponDescription id="crpg_OneHandedBastardSwordAlternative" />
+      <WeaponDescription id="crpg_OneHandedBastardSwordAlternative" />

```

Comment:
why are you doubling the onehanded bastardswordalternative?

Reply:
@namidaka good catch..not sure how that got duped, fixed in the recent push.